BUILD.bazel

```diff
@@ -81,6 +83,10 @@ cc_library(
         ":has_absl": ["GTEST_HAS_ABSL=1"],
         "//conditions:default": [],
     }),
+    features = select({
```

Comment:
Looks like there is no good reason to move this line from previous location.Could you please limit the changes to the actual factual changes to remove the noise.

Reply:
[buildifier](https://github.com/bazelbuild/buildtools/tree/master/buildifier) automatically formats `BUILD` files in the canonical way when it applies fixes, so the new location is the correct one. I can revert to the old format manually, but I don't think it adds a lot of noise, and the next person editing the `BUILD` file will likely do the same changes again.

(Dito other comments)